src/components/starlight/PageTitle.astro

```diff
@@ -0,0 +1,46 @@
+---
+// Starlight doesn't expose it's PAGE_TITLE_ID, so we have to import it like this
+import { PAGE_TITLE_ID } from '~/../node_modules/@astrojs/starlight/constants';
```

Comment:
Can't we wrap the `PageTitle` component from Starlight instead?

Reply:
Oh, right! In this case we can :sweat_smile: